bubble-wrap-style.yaml

```diff
@@ -3174,12 +3175,8 @@ layers:
                     france-a-and-n-road-ref-shields:
                         filter: |
                             function() {
-                                return ((feature.ref && ! feature.shield_text) &&
-                                        /^A .*$/.test(feature.ref) ||
-                                        /^N .*$/.test(feature.ref) ) ||
-                                    ((feature.ref && feature.network == 'e-road') &&
-                                    /^A .*$/.test(feature.ref) ||
-                                    /^N .*$/.test(feature.ref) )
+                                return ((feature.ref && !feature.shield_text) && (/^A .*$/.test(feature.ref) || /^N .*$/.test(feature.ref))) ||
```

Comment:
I'm honestly not entirely sure what we're trying to test for here, so the syntax now works (I think the old one was short-circuiting some logic), but it may not be correct...

Reply:
This is for A-roads and N-roads in Europe (like in France).